### ISSUE TITLE
Episode 3 typos

### ISSUE THREAD

**Issue by djhunter** [UCSBCarpentry/CustomDC-R#1]
There are a couple typos near the beginning of Episode 3:

> Now we can use the functions from the tidyverse package. Let’s use read_csv() to read the data into a data frame (we will learn more about data frames later):
> 
> penguins <- read.csv(path_to_file("penguins.csv"))
> 

The above should be `read_csv` to be consistent with the text.

> If you have a csv file, you may use read_csv to create an object with it. path_to_file() is not nesessary to use read_csv().

Consider rephrasing (also "necessary" is misspelled.) Maybe show the output of `path_to_file("penguins.csv")` first, then do `read_csv(path_to_file("penguins.csv")`, then explain that you could enter any path in `read_csv`.

**Comment by kiblerchris** [comment]
You could also consider just using `data(penguins)` and saving the `read_csv` discussion for later.

**Comment by djhunter** [comment]
Yes, `data(penguins)` would be simpler. Then you could demo `read_csv()` at the end of the lesson after you export `penguins_complete.csv`. Or incorporate it into episode 5.

**Comment by kristi-sara** [comment]
so data(penguins) and leave read_csv to later?

**Comment by apandas** [comment]
Yes. This issue will be resolved by showing `data()` at the beginning of ep 3 with explanatory text, and `read_csv()` will be incorporated into ep 4 under the section "Exporting Data" (with the export of `penguins_complete`). Can consider another placement of the function if it doesn't seem to work well there
